nsls2forge_utils/all_feedstocks.py

```diff
@@ -0,0 +1,99 @@
+import requests
+import github3
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def get_all_feedstocks_from_github(organization):
+    '''
+    Gets all public feedstock repository names from the nsls-ii-forge organization.
+
+
+    Parameters
+    ----------
+    organization : str
```

Comment:
Latest versions of Sphinx may not properly render it if there is a blank space between the argument and colon. Please also change other places below.
```suggestion
    organization: str
```